### DISCUSSION TITLE
Mouse over Temp

### DISCUSSION THREAD

**Discussion by Dgocof** [artisan-roaster-scope/artisan#1266] [General]
I uploaded ver 2.8.4 . When you hover your mouse over your screen in the last versions it would give you time and Bean Temp. In the new version it gives me time and degree rate of rise (Degree of rate of Rise)in the location of the curser. Is there a setting for this? It seems to be picking up info from the right axes which is rate of rise instead of the left axes Temp.

**Comment by MAKOMO** [comment]
Toggle by pressing the d key a feature added some 7 years ago...
